docs/source/reference/prototype.rst

```diff
@@ -7,7 +7,9 @@ The :obj:`@tensorclass` decorator helps you build custom classes that inherit th
 behaviour from :obj:`TensorDict` while being able to restrict the possible entries
 to a predefined set or implement custom methods for your class.
 Like :obj:`TensorDict`, :obj:`@tensorclass` supports nesting, indexing, reshaping,
-item assignment and many more features.
+item assignment and many more features. :obj:`@tensorclass` allows non-tensor entries,
+however all the tensor operations are strictly restricted to tensor attributes. One
```

Comment:
I would expand this a bit. 
- how are supported
- an example of operation
- how nested tensordict and nested tensorclass work

Reply:
Agreed ! Let's raise a separate issue to update the documentation with all the use cases?

Reply:
Added few examples and the edge cases for the documentation